projects/fyde/index.js

```diff
@@ -38,7 +40,22 @@ async function tvl(api) {
 
   // add assets in the deposit escrow
   const tokensEscrow = await api.fetchList({ lengthAbi: 'getAssetListLength', itemAbi: 'assetList', target: DEPOSIT_ESCROW })
-  return api.sumTokens({ tokens: tokensEscrow, owner: DEPOSIT_ESCROW })
+  api.sumTokens({ tokens: tokensEscrow, owner: DEPOSIT_ESCROW })
+
+  const ptBalances = await api.multiCall({
+    calls: PTTOKENS.map(token => ({
+      target: token,
+      params: [YIELDMANAGER]
+    })),
+    abi: 'erc20:balanceOf',
+    withMetadata: true,
+  })
+
+  // tokens are Pendle PT Tokens with weETH underlying asset, price them as weETH which is due at maturity 
```

Comment:
yes, but we have price support for pendle tokens

Reply:
so can you change this to `return api.sumTokens({ owner: YIELDMANAGER, tokens: PTTOKENS})`